### DISCUSSION TITLE
Addon-Probleme mit 3.65.11.20221005

### DISCUSSION THREAD

**Discussion by blackbasket** [OpenCCU/OpenCCU#2036] [🇩🇪 Fragen&Antworten]
Hallo zusammen,

es gibt mit dem hap-homematic Addon reproduzierbare Probleme, sobald das aktuelle Release installiert ist.

Hat jemand von euch eine Idee? Ggf @jens-maus ?

https://github.com/thkl/hap-homematic/issues/575

LG,
Marcel

**Comment by jens-maus** [comment]
Ich weiss von nix. Aber mit dem hap-homematic Addon gibt es leider immer mal wieder gewisse Probleme leider. Musst du entsprechend in dem Ticket das da bei hap-homematic offen ist dann entsprechend erörtern...

**Comment by blackbasket** [comment]
Der Versuch war's wert ;) danke für die schnelle Antwort
